input/OFCB/1/Batched/1.xml

```diff
@@ -20,7 +20,7 @@
             <Source>
                 <outcommod>UOX</outcommod>
                 <outrecipe>UOX</outrecipe>
-                <throughput>1000</throughput>
+                <throughput>1e100</throughput>
```

Comment:
This seems wrong.  Remind me of the units of this, please? Even the number of atoms in a megatonne is only O(10^33).